File: src/main/java/com/task/twinotask/web/MappingController.java
package com.task.twinotask.web;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

// TODO: Change with ViewResolver
@SuppressWarnings("unused")
@Controller
public class MappingController {

	@GetMapping("/login")
	public String login(Model model) {
		return "login";
	}

	@GetMapping({"/", "index"})
	public String homepage(Model model) {
		return "index";
	}

}

File: src/main/java/com/task/twinotask/web/MappingController.kt
package com.task.twinotask.web

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

// TODO: Change with ViewResolver
@Controller
class MappingController {

	@GetMapping("/login")
	fun login(model: Model) = "login"

	@GetMapping("/", "index")
	fun homepage(model: Model) = "index"

}
